src/pages/gdl-in-the-news.js

```diff
@@ -22,24 +21,21 @@ function gdlInNews(props: Props) {
         style={{
           marginLeft: "auto",
           marginRight: "auto",
-          display: "flex",
-          width: "fit-content",
           flex: "1 0 auto",
-          boxShadow: "0px 0px 40px 0px rgba(0, 0, 0, 0.2)",
           height: "fit-content",
         }}
       >
-        <Main style={{ boxShadow: "0px 0px 40px 0px rgba(0, 0, 0, 0.2)" }}>
+        <div className="news-grid">
```

Comment:
For at `flow` skal bli riktig må du si `import * as React from "react"` for da importerer du alt fra react, og kan aksessere det igjennom `React.<something>`. (Det må du gjøre alle steder :) )

Også blir vel ikke props-typingen din helt riktig her?

Reply:
Du kan jo feks se hvordan Frida har gjort det i `allPosts.js` i `gdl-blog`